app/account/components/AccountDetails/components/AvatarUpdate/cropUtils.ts

```diff
@@ -0,0 +1,32 @@
+export const getCroppedImg = async (imageSrc: string, crop: { x: number; y: number; width: number; height: number }, userName: string) => {
+  const avatarMaxSizeInPx = 192
+  const image = new Image()
+  image.src = imageSrc
+
+  return new Promise<File>((resolve, reject) => {
+    image.onload = () => {
+      const canvas = document.createElement("canvas")
+      const ctx = canvas.getContext("2d")
+
+      if (!ctx) {
+        return reject(new Error("Failed to get canvas context"))
+      }
+
+      const size = 129
```

Comment:
Nitpicky, I know, but I think I prefer dimension for clarity? I'll leave it up to you!